### PR TITLE
Introduced new tasks which allow running manual tests without Bender

### PR DESCRIPTION
**Pull request by pomek**
Fixes: #27.

### CODE REVIEW

File: lib/template.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Manual Test</title>
+	<style>
+		div[data-sidebar="true"] {
```
Comment (Reinmar) [inline lib/template.html]: data-sidebar? Can't it be done using an id or class?

**Comment by pomek** [comment]
I introduced missing tests. ATM we cover all the code except ifs in the utils (methods which return config for Karma / Webpack).

image

File: lib/template.html (hunk 1 above)
Comment (Reinmar) [inline lib/template.html]: It does make a difference. This is simply odd, cause there are more standard ways of doing this.

Comment (Reinmar) [inline lib/template.html]: And once it's odd, you start thinking why is it odd.

**Pull request merged**